### PR TITLE
feat: Show more loop tags and fix multisign width

### PR DESCRIPTION
**Pull request by jorgenholme96**
Only show three loop tags with an option to show all.

Multisign and multiverify only takes the relative width, and doesn't take the whole screen.

### CODE REVIEW

File: src/checklist/Checklist.tsx [hunk 1]
```diff
@@ -132,97 +148,142 @@ const Checklist = (props: ChecklistProps): JSX.Element => {
...
-                                  This checklist is signed. Unsign to make
-                                  changes.
-                              </Banner.Message>
-                          </Banner>
-                      )}
-                {checklistDetails ? (
-                    <ChecklistWrapper>
-                        {!isSigned && (
-                            <CheckAllButton
+                {!multiSignOrVerifyIsOpen && (
```
Comment (ErlendLygre) [inline src/checklist/Checklist.tsx:166]: Implement the loop tags section as a separate component - LoopTags.tsx - now that its complexity has increased.

**Comment by Noggling** [comment]
For meg ser dette greit ut men @ErlendLygre får ta en titt på det for sikkerhets skyld ;)

File: src/checklist/Checklist.tsx [hunk 2]
```diff
@@ -35,6 +36,13 @@ const LoopTagWrapper = styled.div`
         margin: 0;
         margin-bottom: 4px;
     }
+    & button {
```
Comment (ErlendLygre) [inline src/checklist/Checklist.tsx:39]: Remove this and use the same button style that we use for "Show more details", defined in CheckItem.tsx on line 36. Now that this button layout will be used in multiple places, please create a new button - "ShowMoreButton.tsx" in the components > buttons folder. This generic button should accept a React.Dispatch<React.SetState<boolean>> function as a prop, and toggle the value. 

Replace your button and the button on CheckItem.tsx line 168 with this new button.

File: src/checklist/ChecklistMultiSignOrVerify.tsx [hunk 3]
```diff
@@ -6,14 +6,10 @@ import { ProcosysApiService } from '../services/procosysApi';
 import { COLORS } from '../style/GlobalStyles';
 
 const Wrapper = styled.div`
```
Comment (ErlendLygre) [inline src/checklist/ChecklistMultiSignOrVerify.tsx:8]: This does need to be position: fixed and fullscreen in the MC app. Lets discuss some solutions for this.

Screenshot 2021-10-07 at 10 32 13
.

File: src/checklist/ChecklistMultiSignOrVerify.tsx [hunk 4]
```diff
@@ -114,7 +110,7 @@ const ChecklistMultiSignOrVerify = ({
 
     return (
         <Wrapper>
-            <p>
+            <p id="multiSignVerifyHeader">
```
Comment (ErlendLygre) [inline src/checklist/ChecklistMultiSignOrVerify.tsx:113]: Please use a ref for this instead of setting ID.

**Review by ErlendLygre — CHANGES_REQUESTED** [review]

File: src/components/buttons/ShowMoreButton.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        align-items: center;
+        & p {
+            margin: 0;
+            color: ${COLORS.mossGreen};
+        }
+    }
+`;
+
+type ShowMoreButtonProps = {
+    handleOnClick: () => void;
```
Comment (ErlendLygre) [inline src/components/buttons/ShowMoreButton.tsx:22]: change this to toggleShowMore: React.Dispatch<React.SetStateAction<boolean>>

**Pull request merged**